charts/simple-app/values.yaml

```diff
@@ -47,6 +47,10 @@ args: []
 # https://kubernetes.io/docs/concepts/storage/volumes/.
 volumes: []
 
+# -- A list of init containers that are run before the main containers start
+# https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+initContainers: []
```

Comment:
the annoying thing with doing it this way is that if there are any differences in `initContainers` among your various values variants, you have to entirely duplicate `initContainers` in each values file.  the issues I ran into in particular are
1. setting `image` explicitly rather than composing it from `repository` and `tag` (super annoying and will make it hard to integrate with existing deploy scripts).
2. setting smaller resource requirements for `local`.  this same issue would manifest for staging.